### DISCUSSION TITLE
logtail商业版和社区版的区别

### DISCUSSION THREAD

**Discussion by sjtuzbk** [alibaba/ilogtail#268] [Help] [answered]
这两个版本在功能上有什么区别吗？

**Comment by messixukejia** [comment, marked as answer]
核心采集、处理能力是一样的，代码完全复用，这部分能力都是经过大规模商用考验的，可以放心使用。

社区版作为一个开源的Agent，需要提供独立运行的能力，所以没有与SLS服务端的管控通道。因此两者的主要差别在于管控能力、自身可观测性上。
具体差异我们在文档里有所体现，https://ilogtail.gitbook.io/ilogtail-docs/about/compare-editions。
核心的几点差异：
1、商业版作为阿里云SLS的官方可观测数据采集器，跟SLS无缝衔接。SLS服务端也提供了机器组、采集配置全局管理等能力，用户通过API、SLS控制台可以方便的进行全局管控。
2、另外，如果业务部署在阿里云ECS、ACK等，也提供了自动安装、一键部署等能力，易用性更高。
3、SLS作为一个可观测平台，商业版也很好的利用了这一点，推出了[CloudLens for SLS](https://help.aliyun.com/document_detail/425764.html)，可以清晰全局展示采集状态、流量状态、处理异常告警等。

当然我们后续也会计划开源一个轻量级的服务端管控方案，可以关注[issue](https://github.com/alibaba/ilogtail/issues/269)。

**Comment by messixukejia** [comment]
是否方便交流下你们使用iLogtail的场景？
